index.js

```diff
@@ -0,0 +1,101 @@
+
+function defineSession(name) {
+  var EditSession = require("ace/edit_session").EditSession;
+  var html = new EditSession(`<!DOCTYPE html>
+  <html>
+  <head>
+  </head>
+
+  <body>
+  </body>
+
+  </html>`, 1);
+  var css = new EditSession(["some", "css", "code here"]);
+  editor.setSession(name);
+}
+
+
+function editorSelect(e) {
+  const selected = e.target;
+  if (selected.className === 'html') {
+    selected.classList.add('selected');
+    selected.nextElementSibling.classList.remove('selected');
+    defineSession('html');
+  } else if (selected.className === 'css') {
+    selected.classList.add('selected');
+    selected.previousElementSibling.classList.remove('selected');
+    defineSession('css');
```

Comment:
if this is calling the css var from above, is the array inside it helpful or what is supposed to be in there?